script/build.js

```diff
@@ -56,7 +56,10 @@ smith.use(collections(BUILD_OPTIONS.collections));
 smith.use(leftRailNavResetLevels());
 smith.use(dateInFilename(true));
 
-smith.use(assets(BUILD_OPTIONS.assets));
+smith.use(assets(BUILD_OPTIONS.appAssets));
```

Comment:
I went through a few different approaches here, but just calling the assets plugin twice with different source folders turned out to be the simplest way to make this work.